__tests__/comments.test.js

```diff
@@ -0,0 +1,64 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe('GET /api/articles/:article_id/comments', () => {
+    let validArticleId;
+
+
+    beforeEach(() => {
```

Comment:
lets not use this to grab the article_id and lets use an article_id we know exists, have a look at the data you are using to seed 😄